test/BackendGeb/Eval/Negative.hs

```diff
@@ -0,0 +1,32 @@
+module BackendGeb.Eval.Negative where
```

Comment:
Since we don't have any negative tests, I think this file should be removed. It contains only dead code.

Reply:
Done.